app/trip/[id]/page.tsx

```diff
@@ -0,0 +1,103 @@
+'use client'
+
+import { Box, Container, useColorModeValue } from '@chakra-ui/react'
+import { useRouter } from 'next/navigation'
+import { PrimaryButton } from '@/components/button'
+import { Loading } from '@/components/loading'
+import { Header, Footer } from '@/components/navigation'
+import { TripDetailsHeader, TripDetailsTabs } from './components'
+import { useTripDetailsQuery, useTripTagsCollectionQuery } from '@generated/api'
+
+export default function TripDetailsPage({
+  params
+}: {
+  params: { id: string }
```

Comment:
Could we use UUID for the parameters? 👀

Reply:
I changed to UUID! 
https://github.com/tabi-memo/frontend/pull/17/commits/c86cda7f0f0a9112492a31b1f66cbee4897ba5a4